### DISCUSSION TITLE
Full album art

### DISCUSSION THREAD

**Discussion by Nolli20** [z-huang/music#1446] [Q&A]
Hi 
I have an honor x8 and I would like to have the album art on the lock screen look like on the iphone

I want that
![iOS-16-beta-3-large-album-art-Lock-Screen-1.jpg](https://github.com/user-attachments/assets/17db3d54-ad5a-4973-b73f-fd27b351742b)

But I have that
![Screenshot_20240823_155139.jpg](https://github.com/user-attachments/assets/cfa83235-4988-4627-aa5d-765f50d929c1)

I don't know if it's possible to have it
Or if It is innnertune that must integrate this feature

**Comment by z-huang** [comment]
As far as I know, Android doesn't have this.

**Comment by Nolli20** [comment]
You can't integrate this feature ?
